src/app/labeling/components/labeling.component.ts

```diff
@@ -1389,7 +1393,31 @@ export class LabelingComponent implements OnInit, OnDestroy {
     }
   }
 
+  addLabelToMarkedTextFromSuggestion(overlayData) {
+    const dataEntry = {
+      startIdx: overlayData.tokenStartIdx,
+      endIdx: overlayData.tokenEndIdx,
+      value: overlayData.value,
+    };
+
+    this.somethingLoading = true;
```

Comment:
when clicking on a weak supervision label, which has an overlapping manual label is not set as expected. But the cursor has a loading spinner attached to it, which isn't removed.

Reply:
saw this too, i will create an issue for this. but only saw this in minor cases